circuits/src/lib.rs

```diff
@@ -753,6 +766,241 @@ pub mod native_helpers {
     ) -> Nullifier {
         compute_poseidon_hash(&[share_commitment, wallet_blinder])
     }
+
```

Comment:
Moved these helpers from `test_helpers` to `native_helpers` to export them outside of the `circuits` crate. Other than that, they are the same